lxd/network/openvswitch/ovn.go

```diff
@@ -1613,19 +1613,48 @@ func (o *OVN) ChassisGroupChassisAdd(haChassisGroupName OVNChassisGroup, chassis
 
 // ChassisGroupChassisDelete deletes a chassis ID from an HA chassis group.
 func (o *OVN) ChassisGroupChassisDelete(haChassisGroupName OVNChassisGroup, chassisID string) error {
-	// Check if chassis group exists. ovn-nbctl doesn't provide an "--if-exists" option for this.
-	output, err := o.nbctl("--no-headings", "--data=bare", "--colum=name,ha_chassis", "find", "ha_chassis_group", fmt.Sprintf("name=%s", string(haChassisGroupName)))
+
+	var chassisUUID string
+
+	// find the _uuid of the received chassis_name
+	output, err := o.nbctl("--format=csv", "--no-headings", "--data=bare", "--column=chassis_name,_uuid", "list", "ha_chassis")
 	if err != nil {
 		return err
 	}
-
+	
 	lines := shared.SplitNTrimSpace(output, "\n", -1, true)
+
+	for _, line := range lines {
+		// node2,a74125a8-b580-4763-b389-11ce2c8c5509
+		fields := strings.Split(line, ",")
+
+		if len(fields) != 2 {
+			return fmt.Errorf("Unrecognised ha_chassis output %q", line)
+		}
+
+		if fields[0] == chassisID {
+			chassisUUID = fields[1]
+			break
+		}
+	}
+
+	if chassisUUID == "" {
+		return fmt.Errorf("Could not find chassis with name %q", chassisID)
+	}
+
+	// Check if chassis group exists. ovn-nbctl doesn't provide an "--if-exists" option for this.
+	output, err = o.nbctl("--no-headings", "--data=bare", "--colum=name,ha_chassis", "find", "ha_chassis_group", fmt.Sprintf("name=%s", string(haChassisGroupName)))
```

Comment:
```suggestion
	output, err = o.nbctl("--no-headings", "--data=bare", "--colum=name,ha_chassis", "find", "ha_chassis_group", "name="+string(haChassisGroupName))
```

`fmt.Sprintf()` is slow so we recently started replacing it with simpler/faster string concatenation.